setup.py

```diff
@@ -0,0 +1,46 @@
+# Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+# Copyright [2016-2020] EMBL-European Bioinformatics Institute
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+from setuptools import setup, find_packages
+
+from wrappers.python3 import eHive
+
+with open(Path(__file__).parent / 'LICENSE') as f:
+    license_ct = f.read()
+
+setup(
+    name='ensembl-hive',
+    package_dir={"": "wrappers/python3/"},
+    packages=find_packages(where='wrappers/python3/'),
+    description="Ensembl Python Base Hive Wrapper",
+    author='Ensembl',
+    author_email='dev@ensembl.org',
+    url='https://www.ensembl.org',
+    download_url='https://github.com/Ensembl/ensembl-py',
+    license=license_ct,
```

Comment:
When reading about `MANIFEST.in`, I've found that there is a [`license_file`](https://setuptools.readthedocs.io/en/latest/setuptools.html#metadata) option. I think we should be using it (and in ensembl-py / ensembl-compara too)

Reply:
That would mean we don't need to do `include LICENSE` in `MANIFEST.in`

Reply:
And since `MANIFEST.in` would only have `include wrappers/python3/README.md`, I wonder if we could find another way of automatically bringing it in via https://packaging.python.org/guides/using-manifest-in/#how-files-are-included-in-an-sdist

Reply:
Yes may be, please could we have a call a some point to discuss all this with our actual voices :-)

Reply:
According to https://packaging.python.org/guides/distributing-packages-using-setuptools/#license `license` shouldn't be the content of the license but the title of the license with the version (and if we don't care about the version, the classifier below would be enough)

Reply:
@JAlvarezJarreta . We should probably do the same in ensembl-compara, see https://github.com/Ensembl/ensembl-hive/blob/features/python-packaging/setup.py#L31